### ISSUE TITLE
How to sort columns after col_split

### ISSUE THREAD

**Issue by stephanmg** [DingWB/PyComplexHeatmap#81]
Hi,

I have a col_split by a `pd.Series` to group into 3 different groups A, B, and C.

The groups A, B, C seem to be automatically sorted ascending.

However I need to sort with a custom order within the groups A, B, C. How is this possible?

**Comment by stephanmg** [comment]
The groups, seem to be sorted also automatically ascending. But the individual columns seem to be sorted arbitrary?

**Comment by DingWB** [comment]
Hi @stephanmg ,

Take a look at:
https://dingwb.github.io/PyComplexHeatmap/build/html/notebooks/advanced_usage.html#Cluster-between-groups-and-cluster-within-groups

**Comment by stephanmg** [comment]
Yes, I forgot that I need to also specify `col_cluster=False`, `col_dendrogram=False` is not enough.

**Comment by stephanmg** [comment]
@DingWB thank you! Is it also possible to remove or reposition the label of a anno_barplot(...)?

Code:
`AB=anno_barplot(...`.

I need to move the label `AB` or remove it in the plot.

**Comment by DingWB** [comment]
How did you control the pad in a normal barplot?
You can control the pad similarly by taking the ax of the `anno_barplot` and doing whatever you want.

**Comment by stephanmg** [comment]
Thanks, what about the label name? Can I somehow include spaces?

**Comment by DingWB** [comment]
Please try.

**Comment by stephanmg** [comment]
No, sorry, I mean in the label`s name. This is a second question, on how to rename the label, e.g. AB for the annotation to `AB variant`

**Comment by DingWB** [comment]
See this docu: https://dingwb.github.io/PyComplexHeatmap/build/html/notebooks/advanced_usage.html#Add-multiple-heatmap-annotations-using-for-loop

You could prepare a dict and pass it to HeatmapAnnotation. You can give whatever key to the dict (including space).

**Comment by stephanmg** [comment]
`legend_kws` with `label` I assume? doesn't work somehow for my anno_barplot. :/

**Comment by stephanmg** [comment]
`row_ha = HeatmapAnnotation(ABadvanced=anno_barplot(histo_data_for_row_ha, colors=bar_color, height=15), verbose=0, label_kws={'label': "# problems solved", 'rotation':0, 'fontsize': 8, 'horizontalignment':'left','verticalalignment':'bottom', 'visible':False}, axis=1)`

**Comment by stephanmg** [comment]
But this will not rename Abadvanced to "# problems solved"... don't know why.

**Comment by WANGchuang715** [comment]
@DingWB 建议对列名进行分组的时候加上 sort=False的参数保持默认的输入顺序

            col_clusters = self.col_split.groupby(cols,sort=False).apply(
                lambda x: x.index.tolist()
            )

**Comment by DingWB** [comment]
非常好的建议，谢谢 @WANGchuang715 
已经在最新的github上更新了
